corehq/blobs/migrate.py

```diff
@@ -269,6 +274,21 @@ def should_process(self, doc):
         return doc.get("external_blobs")
 
 
+class BlobDbBackendExporter(BlobDbBackendMigrator):
```

Comment:
This class should override `_backup_doc` with a no-op. I think there's no need to write anything to the migration log for exports (especially not a full json dump of the doc).